### ISSUE TITLE
Query about pyts 0.7.5

### ISSUE THREAD

**Issue by MANMIS-11** [johannfaouzi/pyts#28]
I am using pyts 0.7.5 version and using GAF to convert time series data into images. When I am using transform(self, X), Shape of  is  defined as:
X : array-like, shape = [n_samples, n_features].

Here, what is the purpose of n_features?

**Comment by hichamjanati** [comment]
If you are referring to the transform method of the GAF object, the latest code (and also in version 0.8) is:

https://github.com/johannfaouzi/pyts/blob/1c0b0c9628068afaa57e036bd157fcb4ecdddee6/pyts/image/gaf.py#L94-L100

So it must the time dimension

**Comment by MANMIS-11** [comment]
Yes. I am referring to the transform method of the GAF object. Is there any documents available where I can understand the mathematical background? Because all the paper that I have read is about one dimensional time series data. 
for example: 
[Encoding time series as images for visual inspection and classification using tiled convolutional neural networks](https://www.aaai.org/ocs/index.php/WS/AAAIW15/paper/viewPaper/10179)

**Comment by johannfaouzi** [comment]
Hi,

The `pyts` API is close to the `scikit-learn`.  In `pyts<=0.7`, the documentation stated that the expected shape was `[n_samples, n_features]`. However, for time series, the term `feature` has a different meaning that for standard machine learning: it refers to the different components of a multivariate time series. To avoid any confusion, the documentation has been changed in `pyts>=0.8`, with `[n_samples, n_features]` being replaced with `[n_samples, n_timestamps]`, but the expected shape is still the same:
* the first dimension is for the samples,
* the second dimension is for the time.

The reference that you gave is the only reference used for the implementation.

Here is the [documentation for GramianAngularField](https://pyts.readthedocs.io/en/latest/generated/pyts.image.GramianAngularField.html#pyts.image.GramianAngularField) on the latest version of master. I encourage you to update to a more recent version of `pyts` (the current version is `0.8.0`, and version `0.9.0` will be released next week). Unfortunately it brings a few changes with no backward compatibility, so you would have to slightly update your code. Anoher mistake I made is that I didn't create a branch for each minor version, so the documentation for version `0.7.5` is no longer available. I could try to see which commit was used for previous versions and create new branches with these commits.

I hope that it helps you a bit, feel free to ask more questions if I wasn't clear.

Johann
